.github/workflows/shadow-study.yml

```diff
@@ -0,0 +1,58 @@
+name: Build and deploy shadow study example
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - "analyses/shadow-study/**"
+      - ".github/workflows/shadow-study.yml"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: analyses/shadow-study
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "18"
+      - run: yarn install --frozen-lockfile
+      - run: yarn typecheck
+      - run: yarn build
+        if: github.ref != 'refs/heads/main'
+
+  deploy:
+    if: github.ref == 'refs/heads/main'
+    needs: test
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: analyses/shadow-study
+
+    permissions:
+      contents: read
+      pages: write
+      id-token: write
+
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "18"
+      - run: yarn install --frozen-lockfile
+      - run: yarn build
+      - uses: actions/configure-pages@v3
+      - uses: actions/upload-pages-artifact@v2
+        with:
+          path: "dist"
```

Comment:
I'm not sure about the pages actions, this is just my best guess about how to set it up. Does it make sense to you along with `working-directory: analyses/shadow-study` @havard-hoiby-adsk ?

Reply:
I'm just gonna try